packages/offix-datastore/src/Model.ts

```diff
@@ -0,0 +1,72 @@
+import { Storage, StoreChangeEvent, EventTypes } from "./storage";
+import { createPredicate, Predicate } from "./predicates";
+
+export interface FieldOptions {
+    // GraphQL type
+    type: string;
+    // GraphQL key
```

Comment:
More info needed.

Reply:
I don't use these fields yet. I only added them because you mentioned them

> * Type
> * Database key
> * Format (which might mean many many fields

I'm using them as placeholders. I can fill in their docs when I use them.